pepper-apis/src/main/java/org/broadinstitute/ddp/pex/TreeWalkInterpreter.java

```diff
@@ -201,12 +202,21 @@ private UnaryOperator extractUnaryOperator(TerminalNode node) {
         }
     }
 
+    private String getUserGuidByUserType(InterpreterContext ictx, TerminalNode node) {
+        if (UserType.OPERATOR.equals(node.getText())) {
+            return ictx.getOperatorGuid();
```

Comment:
If the expression is querying based on operator but our context doesn't have an operator guid, I think we should throw a `PexRuntimeException`.

Reply:
Done